scripts/install_build_deps.sh

```diff
@@ -26,15 +26,21 @@ go install github.com/mikefarah/yq/v4@latest # used to fix OpenAPI spec in scrip
 
 FDB_VERSION=6.3.23
 if [[ "$OSTYPE" == "darwin"* ]]; then
-  FDB_PACKAGE_PATH="$(mktemp -p /tmp/ -u)-FoundationDB-$FDB_VERSION.pkg"
+	if command -v brew > /dev/null 2>&1; then
+		brew install protobuf wget
+	fi
+
+	FDB_PACKAGE_PATH="$(mktemp -u)-FoundationDB-$FDB_VERSION.pkg"
 	FDB_SHA512=d7b89e82dae332af09637543371c58bcaaab2c818a3ea49f56e22587d1a6adfc255e154e6c4feca90f407e37d63d8a3cd2e7cfa0b996c2865c9d74fd5dc1b0ba
-	wget https://github.com/apple/foundationdb/releases/download/$FDB_VERSION/FoundationDB-$FDB_VERSION.pkg -O "$FDB_PACKAGE_PATH"
-	echo "$FDB_SHA512 $FDB_PACKAGE_PATH" | sha512sum -c
+	wget "https://github.com/apple/foundationdb/releases/download/${FDB_VERSION}/FoundationDB-${FDB_VERSION}.pkg" -O "$FDB_PACKAGE_PATH"
+	DOWNLOAD_SHA512=$(shasum -a 512 < ${FDB_PACKAGE_PATH})
+	if [ ${FDB_SHA512} != ${DOWNLOAD_SHA512} ]
+	then
+		echo "Mismatching checksum on downloaded FDB package"
+		exit 1
+	fi
 	sudo installer -pkg "$FDB_PACKAGE_PATH" -target /
 
-	if command -v brew > /dev/null 2>&1; then
-    brew install protobuf
-  fi
 else
   FDB_PACKAGE_PATH="$(mktemp -p /tmp/ -u)-foundationdb-clients_$FDB_VERSION-1_amd64.deb"
```

Comment:
Should we make indentation the same here? It was mixed in the original version also, we should check with @efirs to see what his preference is for editor configs (TAB use appears to be dominant here).

Reply:
TABs, I would say. Mainly because Golang is all tabs too.